sdk/identity/identity/test/manual-integration/AzureFunctions/INSTRUCTIONS.md

```diff
@@ -0,0 +1,50 @@
+# Testing Identity in Azure Functions
+
+## Configuration
+
+Before you begin, you must have the following:
+
+An Azure account with an active subscription.
+
+The Azure Functions Core Tools version 3.x.
+
+One of the following tools for creating Azure resources:
+
+Azure CLI version 2.4 or later.
+
+Azure PowerShell version 5.0 or later.
+
+Node.js, Active LTS and Maintenance LTS versions (8.11.1 and 10.14.1 recommended).
+
+- Prerequisites and Configuration of local env same as this - https://docs.microsoft.com/azure/azure-functions/create-first-function-cli-typescript?tabs=azure-cli%2Cbrowser#configure-your-local-environment
+
+## Prerequisites
+
+In a terminal or command window, run func --version to check that the Azure Functions Core Tools are version 3.x.
+
+Run az --version to check that the Azure CLI version is 2.4 or later.
+
+Run az login to sign in to Azure and verify an active subscription.
+
+- An Azure Key Vault.
+
+### Install Azure Functions
+
+`cd IdentityTest`
+`npm install`
+
+## Give the Azure Functions access to the key vault
+
+1. go to the code in the repo,
+2. follow the instructions here to create RG and deploy to RG - https://docs.microsoft.com/azure/azure-functions/functions-create-first-function-resource-manager?tabs=command-line%2Cazure-cli#deploy-the-template
+
+Note -> if 3 gives error, try 5 first then do 3 to 4
+
+3. On the portal go to the created function app in the resource group - Go to Features -> Managed Identity -> System Assigned -> Set status to On and create a role assignment for Kweyvault and select the keyvault created fro mthe resource name dropdown, you can assign the role of Keyvault Administrator
```

Comment:
For KeyVault we learned we don't actually need a role assignment since we use access policies, not RBAC. So in this case, we don't need the second half of the sentence.

In general though for this section, I'd love if we can provide commands using the az cli instead. I think it would be easier to follow

Reply:
You might be able to set up the system assigned managed identity in the arm template to save a manual step. Try something like this https://github.com/Azure/azure-quickstart-templates/blob/d427f709332f56f44d766df1a87dfc4b58ac80f6/101-functions-managed-identity/azuredeploy.json#L199-L201